### ISSUE TITLE
empty lines inside "methods" must be prohibited

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo#494]
If I add this `yaml` file to `TyposTest` my test fails:

```
[] > app
  hello

    bye
```

The empty line here must lead to parsing error, but it doesn't.

**Comment by yegor256** [comment]
@andreoss can you fix?

**Comment by andreoss** [comment]
@yegor256 Please, assign. Where is `TyposTest`?

**Comment by yegor256** [comment]
@andreoss in eo-parse, just search. It relies on yaml files.

**Comment by andreoss** [comment]
@yegor256 So, this makes single empty line only allowed on top level?
Therefore, this is illegal?
```
[] > ooo
    # This is one
    [] > o-1 /?
                                    <-- empty line
    # This is two
    [] > o2
```

**Comment by yegor256** [comment]
@andreoss empty lines are legal ONLY in front of abstract objects. So, your example is legit.

**Comment by yegor256** [comment]
@andreoss I believe it's fixed, extra test here: c6c47c47
